frontend/src/components/browse-sightings-page/SightingCard.tsx

```diff
@@ -1,22 +1,34 @@
 import React from "react";
-import { Sighting } from "../../clients/apiClient";
+import { Sighting, SightingFromExternalApi } from "../../clients/apiClient";
```

Comment:
Again, this was renamed:
```suggestion
import { Sighting, ExternalSighting } from "../../clients/apiClient";
```